cedar-policy-cli/src/lib.rs

```diff
@@ -94,10 +94,48 @@ pub enum Commands {
     Link(LinkArgs),
     /// Format a policy set
     Format(FormatArgs),
+    /// Translate JSON schema to custom schema syntax and vice versa (except comments)
+    TranslateSchema(TranslateSchemaArgs),
     /// Create a Cedar project
     New(NewArgs),
 }
 
+#[derive(Args, Debug)]
+pub struct TranslateSchemaArgs {
+    /// The direction of translation,
+    ///   JSON -> Natural
+    ///   Natural -> JSON
+    #[arg(long)]
+    pub direction: TranslationDirection,
+    /// Filename to read the schema from.
+    /// If not provided, will default to reading stdin.
+    #[arg(short, long = "schema", value_name = "FILE")]
+    pub input_file: Option<String>,
+}
+
+/// The direction of translation
+#[derive(Debug, Clone, Copy, ValueEnum)]
+pub enum TranslationDirection {
+    /// JSON -> Natural schema syntax
+    JsonToNatural,
+    /// Natural schema syntax -> JSON
+    NaturalToJson,
+}
+
+#[derive(Debug, Clone, Copy, ValueEnum)]
+pub enum SchemaFormat {
+    /// Default human-readable format
+    Human,
```

Comment:
here we use "Human" but in other help text we use "Natural".  We should be consistent, at least in things visible to CLI users in the CLI commands and help